### DISCUSSION TITLE
Sensore con valore legato a secondo sensore

### DISCUSSION THREAD

**Discussion by array81** [indomus/forum#2629] [HUB: Home Assistant]
Ho una stazione meteo che ho collegato con HA.
Tra i vari sensori che ho creato uno mi indica i mm di pioggia cumulati nel tempo, quindi si tratta di un intero che aumenta sempre nel tempo in particolare resta costante quando non piove e aumenta quando piove.
Ora vorrei creare un sensore che mi dica se sta o meno piovendo e ho pensato di farlo funzionare con questa logica: se il sensore dei mm di pioggia aumenta il sensore assume lo stato di ON se invece il sensore dei mm di pioggia non subisce variazioni per un determinato tempo (ad esempio 10 minuti), allora assume lo stato di OFF.
Ovviamente non sarà precisissimo soprattutto per piogge leggere ma per me è sufficiente.
Mi potete dare qualche suggerimento su come poter trattare il ritorno allo stato di OFF legato? Mi ci vorrebbe una specie di timer. E' fattibile farlo a livello di sensore o devo coinvolgere qualche automazione.

**Comment by 50494554524F** [comment]
puoi giocare con history_stats e/o derivative

oppure puoi usare 

`( now() - states['quientità'].last_update ).seconds <= 600`

per intercettare i cambi di stato nei 10 minuti...
